streamlit_dashboard/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas
+altair
+streamlit
+streamlit_option_menu
```

Comment:
Missing the requirements for the dashboard.py file

Reply:
Good spot, adding those now!